frontend/src/v5/ui/v4Adapter/overrides/cards/issues/properties.overrides.ts

```diff
@@ -32,6 +32,10 @@ export default css`
 				padding: 0px 11px;
 			}
 
+			.MuiSelect-select {
+				border: 1px solid ${({ theme }) => theme.palette.base.lightest};
```

Comment:
This one also override the border-color when the dropdown is selected 
![image](https://user-images.githubusercontent.com/43547703/211856259-b6decb8f-3202-44b6-bbcd-c8f902418229.png)